C4/Circulation.pm

```diff
@@ -726,6 +726,13 @@ sub CanBookBeIssued {
     }
 	return ( \%issuingimpossible, \%needsconfirmation ) if %issuingimpossible;
 
+    if ( GetTransfers($item->{itemnumber}) ) {
+                 $issuingimpossible{RESERVED} = 1;
+    }
+
+    return ( \%issuingimpossible, \%needsconfirmation ) if %issuingimpossible;
```

Comment:
This is not needed if you run the above GetTransfers() check before the "# MANDATORY CHECKS - unless item exists, nothing else matters" code